frontend/src/wizards/Argo/ArgoWizard.tsx

```diff
@@ -538,67 +551,71 @@ export function ArgoWizard(props: ArgoWizardProps) {
-                            label="Allow applications to have empty resources"
+                            label={t('Allow applications to have empty resources')}
                         />
                         <WizCheckbox
                             id="apply-out-of-sync-only"
-                            label="Only synchronize out-of-sync resources"
+                            label={t('Only synchronize out-of-sync resources')}
                             path="spec.template.spec.syncPolicy.syncOptions"
                             inputValueToPathValue={booleanToSyncOptions('ApplyOutOfSyncOnly')}
                             pathValueToInputValue={syncOptionsToBoolean('ApplyOutOfSyncOnly')}
                         />
                         <WizCheckbox
                             path="spec.template.spec.syncPolicy.automated.selfHeal"
-                            label="Automatically sync when cluster state changes"
+                            label={t('Automatically sync when cluster state changes')}
                         />
                         <WizCheckbox
                             id="create-namespace"
-                            label="Automatically create namespace if it does not exist"
+                            label={t('Automatically create namespace if it does not exist')}
                             path="spec.template.spec.syncPolicy.syncOptions"
                             inputValueToPathValue={booleanToSyncOptions('CreateNamespace')}
                             pathValueToInputValue={syncOptionsToBoolean('CreateNamespace')}
                         />
                         <WizCheckbox
                             id="validate"
-                            label="Disable kubectl validation"
+                            label={t('Disable kubectl validation')}
                             path="spec.template.spec.syncPolicy.syncOptions"
                             inputValueToPathValue={booleanToSyncOptions('Validate')}
                             pathValueToInputValue={syncOptionsToBoolean('Validate')}
                         />
                         <WizCheckbox
                             id="propagation-policy"
-                            label="Prune propagation policy"
+                            label={t('Prune propagation policy')}
                             path="spec.template.spec.syncPolicy.syncOptions"
                             inputValueToPathValue={checkboxPrunePropagationPolicyToSyncOptions}
                             pathValueToInputValue={checkboxSyncOptionsToPrunePropagationPolicy}
                         >
                             <Select
-                                label="Propogation policy"
-                                options={['foreground', 'background', 'orphan']}
+                                label={t('Propogation policy')}
+                                options={[t('foreground'), t('background'), t('orphan')]}
```

Comment:
Not sure that these should be translated, since they appear to be the actual values for the select options. The actual values will be substituted into YAML, yes?

Note that the options can be an array of Option, rather than just a string, which will allow you to handle translation properly.

Reply:
updated to `[{label:'t(...)', value: '...'}]`